js/app.js

```diff
@@ -0,0 +1,94 @@
+function requestResources() {
+  const request = new XMLHttpRequest();
+  request.addEventListener("load", function() {
+    returnedObject = JSON.parse(this.responseText);
+    console.log(
+      "https://swapi.co/api/" +
+        resourceType.value +
+        "/" +
+        resourceId.value +
+        "/"
+    );
+    console.log(returnedObject);
+    if (returnedObject.detail === "Not found") {
```

Comment:
Instead of using the `returnedObject.detail` use `this.status` and decide what to do based on the status code. The status code is universal while not all APIs will return a detail property.